wiki/installing-windows.md

```diff
@@ -12,8 +12,23 @@ Once the boot menu is open, move the cursor using the arrow keys to the name of
 
 ## Installing
 
-::: danger
-The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. Use `diskpart` to change the partition scheme. ***THIS DELETES ALL DATA ON THE DISK***
+::: warning Caution
+
+The disk that you are trying to install Windows on must be a GPT disk if you are on a UEFI system or MBR if you are on a BIOS system, otherwise it will not let you install. 
+
+Use DiskPart to change the partition scheme. Note: ***THIS MAY DELETE ALL DATA ON THE DISK. MAKE SURE YOU HAVE BACKED UP YOUR DATA BEFORE PROCEEDING***
+
+1. Press Shift + F10 to open a command prompt. From there, type `diskpart`.
+
+2. At the DiskPart prompt, type `list disk` to list all physical disks in your computer. Note the number of the disk you're installing Windows to, and check if there is an asterisk under the GPT column. (An asterisk tells that your system disk is a GPT disk and vice versa.) If you've just switched to UEFI and do not want to delete your data on your MBR system disk, then do not proceed with DiskPart. Instead, exit the DiskPart prompt and type `mbr2gpt /convert /disk:<disk number>` instead. Otherwise, proceed to the next step.
+
+3. Select the target disk. For example, if your system disk has the number 0, type `select disk 0`. After that, type `clean` to wipe the disk.
+
+4. Convert the disk to the corresponding partition scheme, with either `convert mbr` or `convert gpt` to convert the disk to MBR and GPT respectively.
+   
+  ![](./img/installing-windows/gpt.png)
+
+   Exit the command prompt and continue installing.
 :::
```

Comment:
I think a separate section would be more appropriate and would be easier to link to, this is too long for a "caution".

Reply:
Oh, was just done with my work and I didn't read your edited comment. Makes sense, I'll also add another image to demonstrate MBR2GPT.